package.json

```diff
@@ -35,11 +35,11 @@
   },
   "devDependencies": {
     "babel-preset-env": "^1.6.1",
-    "hyperapp": "0.18.0",
-    "jest": "^21.2.1",
+    "hyperapp": "^1.2.1",
+    "jest": "^22.4.3",
     "prettier": "^1.9.2",
-    "rollup": "^0.52.0",
-    "uglify-js": "3.2.2"
+    "rollup": "^0.57.1",
+    "uglify-js": "^3.3.16"
   },
   "peerDependencies": {
     "hyperapp": "0.18.0"
```

Comment:
I think it is necessary to update version.

Reply:
Right

Reply:
@Swizz Could you also update hyperapp: 0.18.0 to the latest?